test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -789,24 +935,30 @@ def test_cgroup_cpuset_and_memsw(self):
         ehost = j.attributes['exec_host']
         tmp_file = filename.split(':')[1]
         tmp_host = ehost.split('/')[0]
-        tmp_out = self.wait_and_read_file(filename=tmp_file, mom=tmp_host)
-        self.wait_and_remove_file(filename=tmp_file, mom=tmp_host)
+        tmp_out = self.wait_and_read_file(filename=tmp_file, mom=self.momA)
+        self.logger.info("output of job is %s" % tmp_out)
+        self.wait_and_remove_file(filename=tmp_file, mom=self.momA)
         self.assertTrue(jid in tmp_out)
         self.logger.info("job dir check passed")
-        self.assertTrue("CpuIDs=0" in tmp_out)
-        self.logger.info("CpuIDs check passed")
-        self.assertTrue("MemorySocket=0" in tmp_out)
-        self.logger.info("MemorySocket check passed")
-        self.assertTrue("MemoryLimit=268435456" in tmp_out)
-        self.assertTrue("MemswLimit=268435456" in tmp_out)
-        self.logger.info("MemoryLimit check passed")
+        if self.paths['cpuacct']:
+            self.assertTrue("CpuIDs=0" in tmp_out)
```

Comment:
Again, is CpuIDs=0 always zero?

Reply:
In this case, it will always be CpuIDs=0 since we know there are no other jobs running.